https://github.com/ethereum/pyethereum/pull/731
commit 2032f07e6e9f5396e5c4c38d684add580629cfb3
Author: AndreMiras <AndreMiras@users.noreply.github.com>
Date:   Mon May 29 23:07:47 2017 +0200

    Update utils.py, fix ImportError
    
    The error was:
    ImportError: No module named warning

diff --git a/ethereum/utils.py b/ethereum/utils.py
index 0137188..d4930b7 100644
--- a/ethereum/utils.py
+++ b/ethereum/utils.py
@@ -14,8 +14,8 @@ import random
 try:
     import secp256k1
 except ImportError:
-    import warning
-    warning.ImportWarning('could not import secp256k1')
+    import warnings
+    warnings.warn('could not import secp256k1', ImportWarning)
     secp256k1 = None
 
 big_endian_to_int = lambda x: big_endian_int.deserialize(str_to_bytes(x).lstrip(b'\x00'))
